package.json

```diff
@@ -1,13 +1,11 @@
 {
   "name": "daffodil-debugger",
-  "displayName": "Daffodil Debugger",
+  "displayName": "Apache Daffodil Debugger Extension",
```

Comment:
Could the name be changed to make it more clear this is about debugging schemas? With this name, it sounds like it's used for debugging Apache Daffodil, which isn't the main focus of the tool. Maybe something like "DFDL Schema Debugger Extension" or "Apache Daffodil DFDL Schema Debugger Extension", though that's a bit long. Maybe this is fine and the description can be used to clarify it's for schema debugging?